buildenv/jenkins/common/pipeline-functions.groovy

```diff
@@ -661,6 +662,25 @@ def retry_and_delay(command, numRetries = 3, waitTime = 60, units = 'SECONDS') {
 }
 
 def setup_pull_request() {
+    // Parse Github trigger comments
+    // For example:
+    /* This is great
+       Jenkins test extended <platform>
+       It was tested*/
+
+    def PARSED_BY_NEWLINE_COMMENT = params.ghprbCommentBody.tokenize(System.lineSeparator())
+    NUM_COMMENTS = PARSED_BY_NEWLINE_COMMENT.size()
+    for (COMMENT in PARSED_BY_NEWLINE_COMMENT) {
+        def comment = COMMENT.toLowerCase().tokenize(' ')
+        if (("${comment[0]}" == "Jenkins") && (("${comment[1]}" == "compile") || ("${comment[1]}" == "test"))) {
```

Comment:
toLowerCase() will never == "Jenkins"

Reply:
I think the lowercase conversion was correct, but we just have to check against `jenkins` instead of `Jenkins`.